syllabus.md

```diff
@@ -0,0 +1,32 @@
+# Syllabus
+
+The following topics relate to code quality.
+
+- [ ] SW Interfaces
+- [x] The build pattern
+- [ ] Anonymous function (lambda)
+  - Closure
+- [ ] The SOLID principles
+  - [ ] Single responsibility principle
+  - [ ] Open-Closed principle
+  - [ ] Liskov Substitution principle
+  - [ ] Interface segregation principle
+  - [ ] Dependency inversion principle
+- [ ] Immutable functions
+- [ ] Naming and comments
+- [ ] Code Structure
+- [ ] Testing
+  - [ ] Test doubles
+  - [ ] Tips for testable code part 1
+  - [ ] Tips for testable code part 2
+  - [ ] Unit, Integration or Functional?
+  - [ ] Test Driven Development
+- [ ] Source control
+  - [ ] The commit scope
+  - [ ] Keep the history clean (rebase)
+  - [ ] Merge and split a commit
+- [ ] CI/CD
```

Comment:
Please consider discussing the importance of a (quick) feedback loop.

Reply:
I think it should be part of any talk about unit tests and CI/CD.
I do not see how it can be its own topic/lecture.